packages/components/src/stories/NumberShower.stories.tsx

```diff
@@ -2,8 +2,11 @@ import type { Meta, StoryObj } from "@storybook/react";
 
 import { NumberShower } from "../components/NumberShower.js";
 
+/**
+ * The number shower is a simple component to display a number.
+ * It uses the symbols "K", "M", "B", and "T", to represent thousands, millions, billions, and trillions. Outside of that range, it uses scientific notation.
```

Comment:
Kudos for the extra comment. Separate: I imagine we'll want something more powerful for extreme orders of magnitude, and tiny numbers, which we get a lot now.

Reply:
It's a copy-paste from the old mdx, I didn't write this comment. But wrote some others in other stories.

Note that this is not just a source code comment! It's JSDoc that is displayed in the Storybook.